applications/app/views/fragments/crosswords/crosswordContent.scala.html

```diff
@@ -36,6 +38,11 @@ <h3 class="crossword__clues-header">Down</h3>
                     </div>
                 </div>
             </div>
+            @if(CommercialSwitch.isSwitchedOn && shouldShowAds(crosswordPage)) {
```

Comment:
I wonder whether `shouldShowAds` should hold the switch state, so that it doesn't get forgotten.  But not really part of this PR